packages/react-next/etc/react-next.api.md

```diff
@@ -1110,7 +1118,361 @@ export { ThemeProviderProps }
 export const Toggle: React.FunctionComponent<IToggleProps>;
 
 // @public (undocumented)
-export const ToggleBase: React.FunctionComponent;
+export const ToggleBase: import("@fluentui/react-compose").ComponentWithAs<"div", IToggleProps>;
+
+// @public (undocumented)
+export const useToggle: (props: IToggleProps, options: ComposePreparedOptions<{}>) => {

```

Comment:
wow. could you give an `any` return type to remove the crazy return type for now?

Reply:
Done